Makefile

```diff
@@ -57,7 +57,7 @@ PAGES_DEPS := \
 	$(wildcard $(SOURCE)/site/search/*) \
 	$(wildcard $(SOURCE)/site/templates/*)
 
-$(STAGING)/site/index.html $(STAGING)/docs.json: $(PAGES_DEPS)
+$(STAGING)/site/index.html $(STAGING)/docs.json $(STAGING)/docs.api.json: $(PAGES_DEPS)
```

Comment:
I don't think we need to write `docs.json` anymore, or don't need both `docs.json` and `docs.api.json`. The only thing that consumes `docs.json` is search.